pkg/api/v1/atlasbackupschedule_types_test.go

```diff
@@ -0,0 +1,81 @@
+package v1
+
+import (
+	"testing"
+
+	"github.com/go-test/deep"
+	"go.mongodb.org/atlas/mongodbatlas"
+
+	"github.com/mongodb/mongodb-atlas-kubernetes/pkg/util/toptr"
+)
+
+func Test_BackupScheduleToAtlas(t *testing.T) {
+	testData := []struct {
```

Comment:
I always suggest individual subtests in favor of table tests in cases with dense data and reduced readability.

Reply:
A matter of taste I think, but converted to a sub-test